contracts/internals/claimable.sol

```diff
@@ -31,7 +31,7 @@ contract Claimable {
     /// @param _to is the recipient's address.
     /// @param _asset is the address of an ERC20 token or 0x0 for ether.
     /// @param _amount is the amount to be transferred in base units.
-    function _claim(address _to, address _asset, uint _amount) internal {
+    function _claim(address payable _to, address _asset, uint _amount) internal {
```

Comment:
Does setting it to `payable` mean that you can use `.transfer()` on it?

Reply:
it's a 0.5 feature, it means that if it is a contract it has a payable function somewhere